docs/user_guide/plotting.md

```diff
@@ -1,11 +1,341 @@
+[config objects](building_blocks/config_objects) section and the column corresponding to the expression
+statement needs to be stored either after the {py:class}`~columnflow.tasks.reduction.ReduceEvents`
+For each of the category given with the ```--categories``` parameter, one plot will be produced.
+A detailed guide on how to implement categories in Columnflow is given in the
+[categories](building_blocks/categories) section.
+
+To define, which processes and datasets to consider when plotting, you can use the ```--processes```
+and ```--datasets``` parameter. When only processes are given, all datasets corresponding to the
+requested processes will be considered. When only datasets are given, all processes in the config
+will be considered.
+The ```processes``` parameter can be used to change the order of processes in the stack and the legend
+(try for example ```--processes st,tt``` instead) and to further distinguish between sub-processes
+(e.g. via ```--processses, tt_sl,tt_dl,tt_fh```).
+
+:::{dropdown} Note! Do not add the same dataset via multiple processes!
+At the time of writing this documentation, there is still an issue present that histograms corresponding
+to a dataset can accidentally be used multiple times. For example, when adding ```--processes tt,tt_sl```,
+the events corresponding to the dataset ```tt_sl_powheg``` will be displayed twice in the resulting
+plot.
+:::
+
+
+## Customization of plots
+
+There are many different parameters implemented that allow customizing the style of a plot. A short
+description to all plotting parameters is given in the [Plotting tasks](../task_overview/plotting.md).
+In the following, a few exemplary task calls are given to present the usage of our plotting parameters.
+
+Per default, the PlotVariables1D task creates one plot per variable with all Monte Carlo backgrounds being included
+in a stack and data being shown as separate points. The bottom subplot shows the ratio between signal
+and all processes included in the stack and can be disabled via the ```--skip_ratio``` parameter.
+To change the text next to the label, you can add the ```--cms-label``` parameter.
+To compare shapes of multiple processes, you might want to plot each process separately as one line.
+To achieve this, you can use the ```unstack``` option of the ```--process-settings``` parameter. This
+parameter can also be used to change other attributes of your process instances, such as color, label,
+and the scale. To better compare shapes of processes, we can normalize each line with the
+```--shape-norm``` parameter. Combining all the previously discussed parameters might lead to a task
+call such as
+
+```shell
+law run cf.PlotVariables1D --version v1 --processes tt,st --variables n_jet,jet1_pt \
+    --skip-ratio --shape-norm --cms-label simpw \
+    --process-settings "tt,unstack,color=#e41a1c:st,unstack,label=Single Top"
+```
```

Comment:
The `--cms-label simpw` might be a little misleading since we haven't discussed the keywords for the cms-label here. We might want to include e.g. a table of keywords and how they are translated for the final plot

Reply:
good point, added with the next commit